modules/openapi-generator/src/main/resources/python/api_doc_example.mustache

```diff
@@ -8,22 +8,26 @@ from pprint import pprint
 {{#hasAuthMethods}}
 # Defining host is optional and default to {{{basePath}}}
 configuration.host = "{{{basePath}}}"
-# Create an instance of the API class
-api_instance = {{{packageName}}}.{{{classname}}}({{{packageName}}}.ApiClient(configuration))
-{{#allParams}}{{paramName}} = {{{example}}} # {{{dataType}}} | {{{description}}}{{^required}} (optional){{/required}}{{#defaultValue}} (default to {{{.}}}){{/defaultValue}}
-{{/allParams}}
+# Enter a context with an instance of the API client
```

Comment:
Aren't these blocks `{{#hasAuthMethods}}`/`{{^hasAuthMethods}}` identical except for setting 
 configuration.host?
Please move the identical block contents  underneath the `{{/hasAuthMethods}}` on line 25